### PR TITLE
Logging context data (custom attributes) documentation

### PR DESCRIPTION
**Pull request by nr-ahemsath**
This PR updates .NET agent documentation (agent configuration and logs-in-context guide) for upcoming support for logging context data (AKA custom attributes).

This feature has not been released yet, but will be released in agent version 10.4.0.  Please do not merge this PR to `develop` until we release 10.4.0, sometime next week (week of 12/5).

This PR is initially in draft status to solicit feedback from the rest of the .NET agent team before requesting docs team review.

### CODE REVIEW

File: src/content/docs/apm/agents/net-agent/configuration/net-agent-configuration.mdx [hunk 1]
```diff
@@ -3212,14 +3212,17 @@ There are three main sub-features:
```
Comment (chynesNR) [inline src/content/docs/apm/agents/net-agent/configuration/net-agent-configuration.mdx:3212]: It's hidden in the diff, but I think it should say "there are four main sub-features" now

File: src/content/docs/apm/agents/net-agent/configuration/net-agent-configuration.mdx [hunk 2]
```diff
@@ -3212,14 +3212,17 @@ There are three main sub-features:
 
 1. Metrics: Collect metrics on the total number of log lines written per harvest cycle (`Logging/lines`), as well as the number of log lines written at particular logging levels (for example, `Logging/lines/ERROR`).
 2. Log forwarding: When enabled, the agent will capture log data and send it to New Relic.
+  1. Context data (custom attributes): when enabled, the agent will capture and forward any custom log attributes.  The "include" and "exclude" elements are comma-separated lists of attribute names to include or exclude, following the [same rules](docs/apm/agents/net-agent/attributes/enable-disable-attributes-net/#attruls) as other agent attribute configuration.  They are both empty by default, which results in all log context data being captured and forwarded.
```
Comment (chynesNR) [inline src/content/docs/apm/agents/net-agent/configuration/net-agent-configuration.mdx:3215]: The numbering looks wrong, but the preview is correct, so I'm not sure if it needs fixing

**Review by chynesNR — APPROVED** [review]
Ignore my earlier comments, I misread.

**Review by nbaenam — DISMISSED** [review]

**Review by chynesNR — APPROVED** [review]

**Review by rhetoric101 — APPROVED** [review]
@nr-ahemsath, thanks for adding these changes!

**Pull request merged**